commands/audit/auditparams.go

```diff
@@ -7,14 +7,17 @@ import (
 
 type AuditParams struct {
 	xrayGraphScanParams *services.XrayGraphScanParams
-	workingDirs         []string
-	installFunc         func(tech string) error
-	fixableOnly         bool
-	minSeverityFilter   string
+	// common params to all scan routines
+	commonGraphScanParams *CommonGraphScanParams
+	workingDirs           []string
+	installFunc           func(tech string) error
+	fixableOnly           bool
+	minSeverityFilter     string
 	*xrayutils.AuditBasicParams
 	xrayVersion string
 	// Include third party dependencies source code in the applicability scan.
 	thirdPartyApplicabilityScan bool
+	numOfParallelScans          int
```

Comment:
```suggestion
	threads          int
```
this parameter control not only the number of parallel scans but parallel tasks - the number of threads in the command (downloading the AM also done in separeted routine)